README.md

```diff
@@ -847,10 +245,10 @@ Run it with
 ./emissary topology
 ```
 
-### Running server with SSL
+#### Running server with SSL
 
 The keystore and keystore password are in the emissary.client.EmissaryClient-SSL.cfg file.  Included and configured
-by default is a sample keystore you can use for testing this functionality.  It is not recommended to use in 
+by default is a sample keystore you can use for testing this functionality -- it is not recommended using in 
```

Comment:
still sounds like a grammar problem to me.  We do not recommend using? It is not recommended for use in?